mhkit/tidal/io/__init__.py

```diff
@@ -1,2 +1,6 @@
+"""
+This module initializes the tidal I/O package, importing necessary submodules.
```

Comment:
Copying more descriptive sentence from our API docs:
```suggestion
The io submodule contains functions to load NOAA and Delft3D tidal velocity and direction data.